.github/workflows/ci.yml

```diff
@@ -12,24 +12,39 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: opendistroforelasticsearch/security-maven:v1
 
     steps:
 
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11.0.x
+
     - name: Checkout security
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
 
     - name: Checkstyle
       run: mvn checkstyle:checkstyle
-      
+
+    - name: prepare openssl
+      run: |
+        mkdir -p tmp/build-openssl && cd tmp/build-openssl
+        curl -OL https://ftp.openssl.org/source/openssl-1.0.1u.tar.gz
```

Comment:
Also the workflow runs on Ubuntu image. Can we not directly use openssl and set OPENDISTRO_SECURITY_TEST_OPENSSL_OPT=true for the test ?

Reply:
Does it come with a compatible openssl version? My understanding is that netty version used by the plugin requires openssl 1.0.x.

Reply:
Github image does not come with openssl preinstalled: https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md